src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -607,9 +503,14 @@ public void ValidatePermissionsInConfig(RuntimeConfig runtimeConfig)
         /// </summary>
         /// <param name="permission"></param>
         /// <returns>True/False</returns>
-        public bool IsValidDatabasePolicyForAction(PermissionOperation permission)
+        public bool IsValidDatabasePolicyForAction(EntityAction permission)
         {
-            return !(permission.Policy?.Database != null && permission.Name == Config.Operation.Create);
+            if (permission.Action is EntityActionOperation.Create)
+            {
+                return string.IsNullOrEmpty(permission.Policy?.Database);
```

Comment:
Would we want to check for whitespaces as well?

Reply:
Added, along with test coverage